README.md

```diff
@@ -1 +1,104 @@
-# compatibility-api-java
\ No newline at end of file
+# Java Wrapper for SignalWire Compatibility APIs
+
+An Unofficial Java API wrapper that allows for the rapid development and integration of Java-based applications with the SignalWire
+API.
+
+> ⚠️ Disclaimer:
+> 
+> The libraries in this repository are NOT supported by SignalWire. SignalWire will _not_ answer support questions about the code in this repository, and is not responsible for fixing issues.
+
+This library automates the process of consuming the Signalwire API by providing a wide range of helper methods that
+allow for quick development and testing.
+
+## Documentation
+
+The documentation for the SignalWire API can be
+found [here](https://developer.signalwire.com/compatibility-api/reference/overview)
+
+## Getting Started
+
+### Dependencies
+
+```xml
+<dependency>
+    <groupId>io.github.signalwire-community</groupId>
+    <artifactId>compatibility-api-java</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+## Usage
+
+Initialize `SignalWireClient`
+
+```java
+SignalWireClient client=new SignalWireClient(projectId,apiToken,spaceName);
+```
+
+This class give you access to the `voice`, `message`, `accounts`, `phonenumber` repository
+
+Get your project Id and API token from your signalWire space. for more information
+click [here](https://developer.signalwire.com/apis/docs/navigating-your-space)
+
+### Sending a Message
+
+```java
+    NewMessageResponse response=client.message().create(to,from,body);
```

Comment:
Could you make sure that all the code blocks in the README are properly indented to the left?

I also suggest a space before and after the ` = ` for all code blocks

Reply:
I will also make this change